drivers/timer/nrf_grtc_timer.c

```diff
@@ -456,7 +452,9 @@ uint32_t sys_clock_elapsed(void)
 		return 0;
 	}
 
-	return (uint32_t)(counter_sub(counter(), last_count) / CYC_PER_TICK);
+	last_elapsed = (uint32_t)counter_sub(counter(), last_count);
```

Comment:
Is it safe to assume that elapsed ticks will always fit into a 32-bit variable (for 1 MHz clock, it's maximum 4294 seconds)?